ia/src/process.py

```
@@ -23,3 +23,6 @@ def iaProcess(argv: list[str]) -> None:
         # decrypted = get_decrypted_message(crypted)
         # print("sentence decrypted: ", decrypted)
         # Parser = ia_parser(argv)
+        Parser = ia_parser(argv)
+        return Parser
```

Ruff (0.11.9)

Unnecessary assignment to `Parser` before `return` statement

Remove unnecessary assignment

(RET504)